src/Microsoft.CodeAnalysis.Analyzers/UnitTests/Microsoft.CodeAnalysis.Analyzers.UnitTests.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
     <ServerGarbageCollection>true</ServerGarbageCollection>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="$(MicrosoftCodeAnalysisVersionForTests)" />
```

Comment:
😕 Why is this changing?

Reply:
@sharwell This is done in multiple unit test projects. The effect AFAIK is using a recent compiler version for compiling the code. It's unnecessary currently, but I think it's desired. Let me know if you want to revert that.